mappings/net/minecraft/block/MapColor.mapping

```diff
@@ -1,60 +1,69 @@
 CLASS net/minecraft/class_3620 net/minecraft/block/MapColor
 	COMMENT Represents the surface color of a block when rendered from the {@link net.minecraft.client.gui.MapRenderer}.
-	FIELD field_15976 CLAY Lnet/minecraft/class_3620;
+	COMMENT Color names referencing a material or object refer to their vanilla MineCraft textures, not their real-world counterparts, eg. "emerald green."
```

Comment:
```suggestion
	COMMENT Color names refer to a material or an object which refers to their vanilla Minecraft textures, not their real-world counterparts, eg. "emerald green."
```

Even after my suggestion I do believe this needs another look from someone else.